rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -83,8 +83,13 @@ def _init_db_with_target_version(
         'rotkehlchen.db.dbhandler.DB_SCRIPT_CREATE_TABLES',
         new='',
     )
+    no_sanity_check = patch(
+        'rotkehlchen.db.dbhandler.db_schema_sanity_check',
+        new=lambda **_: None,
+    )
     with ExitStack() as stack:
         stack.enter_context(target_patch(target_version=target_version))
+        stack.enter_context(no_sanity_check)
```

Comment:
why do you skip the sanity check? Because the end state is not the state of the `DB_SCRIPT_CREATE_TABLES`?

Reply:
Yeah, exactly